src/createNodeResolvePreprocessor.js

```diff
@@ -18,35 +18,54 @@ exports.filePathToUrlPath = (filePath, basePath, urlRoot, proxyPath) => {
 };
 
 exports.createNodeResolvePreprocessor = (karmaConfig, logger) => {
+  let moduleResolve;
+  let findUp;
   const log = logger.create("preprocessor:node-resolve");
 
-  const resolvePath = (modulePath) => {
-    if (!exports.isLocalPath(modulePath)) {
+  const requirePackageJson = (packageName) => {
+    try {
+      return require(join(packageName, "package.json"));
+    } catch {}
+
+    return require(findUp("package.json", { cwd: dirname(require.resolve(packageName)) }));
```

Comment:
This would error on resolving the packageName if it doesn't exist I believe.  I think we can remove findUp and this would be good.

Reply:
Sure, I removed `find-up`, if I ever run into a case where it's useful I'll bring it back!